scripts/github/release-artifacts.sh

```diff
@@ -9,6 +9,7 @@ Options:
   --tag           <tag>             The release tag.
   --workflow      <workflow>        The workflow which builds/archives the artifacts.
   --repo-org      <repo-org>        The repo's owner organization.
+  --binary-repo   <binary-repo>     The repo where binary is built.
```

Comment:
Given the arg above uses `repo-org`, let's call this one just `repo`.

```suggestion
  --repo   <binary-repo>     The repo where binary is built.
```